### PR TITLE
Remove sweeper for logging_logView

### PR DESCRIPTION
**Pull request by hao-nan-li**
Remove sweeper for logging_logView. This resource has a parent resource that wipes it out. Therefore we disable the sweeper for the child resource.

If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Generated Terraform providers](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/generate-providers.md), and ran [`make test` and `make lint`](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-unit-tests) in the generated providers to ensure it passes unit and linter tests.
- [ ] [Ran](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/run-provider-tests.md) relevant acceptance tests using my own Google Cloud project and credentials (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: mmv1/products/logging/LogView.yaml [hunk 1]
```diff
@@ -24,6 +24,7 @@ references: !ruby/object:Api::Resource::ReferenceLinks
   guides:
     'Official Documentation': 'https://cloud.google.com/logging/docs/apis'
   api: 'https://cloud.google.com/logging/docs/reference/v2/rest/v2/projects.locations.buckets.views'
+skip_sweeper: true
```
Comment (rileykarson) [inline mmv1/products/logging/LogView.yaml:27]: Can you comment with the reasoning inline?

**Review by rileykarson — APPROVED** [review]

File: mmv1/products/logging/LogView.yaml (hunk 1 above)
Comment (hao-nan-li) [reply mmv1/products/logging/LogView.yaml:28]: Done

**Comment by rileykarson** [comment]
I wonder if we could reliably identify child resources, rather than use `skip_sweeper` on 'em. We can't _100%_ rely on just nonstandard URL fields, but it could be close.

**Comment by hao-nan-li** [comment]
> I wonder if we could reliably identify child resources, rather than use `skip_sweeper` on 'em. We can't _100%_ rely on just nonstandard URL fields, but it could be close.

Let me see if I can find any pattern going through the other collected sweepers.

**Pull request merged**